controllers/nutanixcluster_controller.go

```diff
@@ -264,7 +264,7 @@ func (r *NutanixClusterReconciler) reconcileNormal(rctx *nctx.ClusterContext) (r
 
 func (r *NutanixClusterReconciler) reconcileCategories(rctx *nctx.ClusterContext) error {
 	klog.Infof("%s Reconciling categories for cluster %s", rctx.LogPrefix, rctx.Cluster.Name)
-	defaultCategories := GetDefaultCAPICategoryIdentifiers(rctx.Cluster.Name)
+	defaultCategories := GetDefaultCAPICategoryIdentifiers(rctx.Cluster.Name, string(rctx.Cluster.GetUID()))
```

Comment:
The cluster UID changes after clusterctl move. As a result different category values (uuids) will be assigned after performing the move.

Reply:
we will have to skip using clusteruuid in this case and only keep clustername based category